app/routes/app+/showcase.tsx

```diff
@@ -0,0 +1,270 @@
+      <section className="space-y-2 max-w-3xl mb-16">
+        <h1 className="text-3xl font-semibold">Showcase</h1>
+        <div>
+          <p className="text-lg text-sky-500">Discover top brainstorm and analytics submissions across the ecosystem</p>
+          <p className="text-gray-500 text-sm">Quickly surface winning submissions from any Marketplace</p>
+        </div>
+      </section>
+      <section className="flex flex-col-reverse md:flex-row space-y-reverse gap-y-7 gap-x-5">
+        <div className="space-y-3 flex-1">
+          <p className="font-semibold">Top Submissions</p>
+          <hr className="bg-gray-200" />
+          <SubmissionsListView submissions={submissions} projects={projects} />
+          <Link to={`?${searchParams.toString()}`} className="text-md text-stone-500">
+            View 5 more
+          </Link>
+        </div>
+        <aside className="md:w-1/4 lg:w-1/5 pt-11">
+          <SearchAndFilter projects={projects} laborMarkets={laborMarkets} />
+        </aside>
+      </section>
+    </Container>
+  );
+}
+
+function SearchAndFilter({ projects, laborMarkets }: { projects: Project[]; laborMarkets: LaborMarketDoc[] }) {
+  const submit = useSubmit();
+  const formRef = useRef<HTMLFormElement>(null);
+
+  const handleChange = () => {
+    if (formRef.current) {
+      submit(formRef.current, { replace: true });
+    }
+  };
+
+  return (
+    <ValidatedForm
+      formRef={formRef}
+      method="get"
+      noValidate
+      validator={validator}
+      onChange={handleChange}
+      className="space-y-3 p-4 border border-gray-300/50 rounded-lg bg-blue-300 bg-opacity-5 text-sm sticky top-0"
+    >
+      <ValidatedInput
+        placeholder="Search address"
+        name="q"
+        size="sm"
+        iconRight={<MagnifyingGlassIcon className="w-5 h-5 text-gray-400" />}
+      />
+
+      <h3 className="font-semibold">Filter</h3>
+      <Checkbox name="type" label="Analyze" value="analyze" />
+      <Checkbox name="type" label="Brainstorm" value="brainstorm" />
+
+      <Field>
+        <Label>Chain/Project</Label>
+        <ValidatedCombobox
+          name="project"
+          onChange={handleChange}
+          placeholder="Select option"
+          size="sm"
+          options={projects.map((p) => ({ value: p.slug, label: p.name }))}
+        />
+      </Field>
+
+      <Field>
+        <Label>Score</Label>
+        <ValidatedSelect
+          name="score"
+          size="sm"
+          onChange={handleChange}
+          placeholder="Select option"
+          options={[
+            { label: "Great", value: "100" },
+            { label: "Good", value: "75" },
+          ]}
+        />
+      </Field>
+
+      <Field>
+        <Label>Timeframe</Label>
+        <ValidatedSelect
+          name="timeframe"
+          size="sm"
+          onChange={handleChange}
+          placeholder="Select option"
+          options={[
+            { label: "Past 24 hours", value: oneUnitAgo("day").toISOString() },
```

Comment:
I think the value here will have to be an enum or otherwise the dropdown doesn't work properly (since the time changes). Then on the backend side you can map the enum to the correct calculation.

you could even have the value be `"day" | "week" | "month"` and the backend would just be `oneUnitAgo(timeframe).toISOString()`

hope that makes sense